_slides/09-samkrishnan.md

```diff
@@ -1,9 +1,9 @@
----
-layout: slide
-title: "Welcome to our slide deck!"
----
-
-Caption here
-
+﻿---
+layout: slide

```

Comment:
This is a prove we can approve Pool Request

Reply:
reviewed and approved